include/librealsense2/dds/dds-client.cmake

```diff
@@ -0,0 +1,14 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+set(DDS_CLIENT_FILES 
+    "${CMAKE_CURRENT_LIST_DIR}/dds-participant.h"
+    "${CMAKE_CURRENT_LIST_DIR}/dds-participant.cpp"
+)
+
+source_group( "DDS" FILES ${DDS_CLIENT_FILES} )
```

Comment:
This make a `DDS` flat folder  in parallel to librealsese/include  right?
librealsense/
                       DDS/
                       include/

You wanted that the hierarchy will be the same as the filesystem one no?

Reply:
This folder will have both sources and includes.
But your point is good -- we could put it under `include/DDS` but I kinda like it...
![image](https://user-images.githubusercontent.com/54848706/169679223-832931c7-7366-4836-96d3-abc515ce7e92.png)
What do you think?

Reply:
Not a big deal, it's only on DDS. 😀